src/framework/global/modularity/ioc.h

```diff
@@ -28,30 +28,30 @@
 
 #define INJECT(Interface, getter) \
 private: \
-    mutable std::shared_ptr<Interface> m_##getter = nullptr; \
+    mutable std::shared_ptr<Interface> mutable_##getter = nullptr; \
```

Comment:
I would not want to rename this variable - it is essentially a regular class field, just like other class fields, I would like them to look the same `m_...`

Reply:
and here the `m` seems to have a different meaning than ther other `m`s?

Reply:
Here the letter `m_` means member, that is, it simply matches the code style of the class member declaration. 
https://github.com/musescore/MuseScore/wiki/CodeGuidelines

Reply:
OK, if you prefer this PR (over #17667) and its change from `s_` to `static_` but want to keep that `m_`, I can certainly do that.